presto-main/src/main/java/com/facebook/presto/execution/CreateTableTask.java

```diff
@@ -78,11 +81,11 @@ public String explain(CreateTable statement, List<Expression> parameters)
     @Override
     public ListenableFuture<?> execute(CreateTable statement, TransactionManager transactionManager, Metadata metadata, AccessControl accessControl, QueryStateMachine stateMachine, List<Expression> parameters)
     {
-        return internalExecute(statement, metadata, accessControl, stateMachine.getSession(), parameters);
+        return internalExecute(statement, transactionManager, metadata, accessControl, stateMachine.getSession(), parameters);
     }
 
     @VisibleForTesting
-    public ListenableFuture<?> internalExecute(CreateTable statement, Metadata metadata, AccessControl accessControl, Session session, List<Expression> parameters)
+    ListenableFuture<?> internalExecute(CreateTable statement, TransactionManager transactionManager, Metadata metadata, AccessControl accessControl, Session session, List<Expression> parameters)
```

Comment:
Please revert this after making the change to use `Metadata` as describe above. Though feel free to add another commit to make it package private (which it should have been before)